tcms/rpc/api/testcase.py

```diff
@@ -337,6 +336,7 @@ def remove(query):
         :param query: Field lookups for :class:`tcms.testcases.models.TestCase`
         :type query: dict
         :return: None
+        :rtype: NoneType
```

Comment:
In many instances above the :return: None reference has been remove but here you specify the return type. Why the difference ? IMO all of these should be consistent.

Reply:
Apologies, I had switched over later on and missed that I had done this already. I've removed the reference